src/main/java/org/openstreetmap/atlas/checks/validation/points/OrphanNodeCheck.java

```diff
@@ -18,8 +18,8 @@
  */
 public class OrphanNodeCheck extends BaseCheck<Long>
 {
-    private static final List<String> FALLBACK_INSTRUCTIONS = Arrays
-            .asList("Node with OSM ID {0} is an orphan, no tags and not connected to any ways.");
+    private static final List<String> FALLBACK_INSTRUCTIONS = Arrays.asList(
+            "Node with OSM ID {0,number,#} is an orphan, no tags and not connected to any ways.");
```

Comment:
Nit: Was looking at this check recently. The instruction reads strangely. Maybe "... is an orphan - it doesn't have any tags, is not connected to any ways and is not part of a Relation".